psi4/src/bin/psi4_main/CMakeLists.txt

```diff
@@ -28,15 +28,15 @@ set(sources_list export_psio.cc
 )
 
 set(PLUGIN_INCLUDES "-I${PSI4_ROOT} -I${Boost_INCLUDE_DIR} -I${PYTHON_INCLUDE_DIR} ")
-message(STATUS "${PLUGIN_INCLUDES}")
+
 add_definitions(-DPSI_TOP_SRCDIR=${PROJECT_ROOT}
     -Drestrict=${RESTRICT_KEYWORD}
     -DPLUGIN_CXX=${CMAKE_CXX_COMPILER}
     -DPLUGIN_INCLUDES=${PLUGIN_INCLUDES}
     -DPLUGIN_FLAGS=${CMAKE_CXX_FLAGS}
```

Comment:
Should CXX_FLAGS also be quoted, in case multiple flags are specified?  I see that's what you did with LINK_FLAGS below.